tests/input/credential-context-combo4-fail.json

```diff
@@ -1,7 +1,7 @@
 {
   "@context": [
     "https://www.w3.org/ns/credentials/v2",
-    "urn:not:a:url"
+    "//not-a-url/contexts/example/v1"
```

Comment:
Why are all these not-URLs using a "//..." form?  Does that test some edge case code?  The simple string "not-a-url" would also fail.  That might be clearer and is more similar to the previous URN strings.  In practice we have seen attempts to use simple strings (like UUIDs) in place of URL ids, so it might be good to have a check like that in at least one test.

Reply:
I actually kind of like a url failing because it has no protocol, but I went ahead and made the change here: https://github.com/digitalbazaar/vc-data-model-2-test-suite/pull/7/commits/b1b9517aed75add89e339a1d3d1ed3fb9b0d044c

Reply:
If it's a concern, tests can be added for variations of failure inputs.  I see you did "not-a-url/foo/bar".  I meant that the literal shorter "not-a-url" string would be enough to fail.  But it's fine either way.